Mage.Tests/src/test/java/org/mage/test/cards/single/woe/AshiokWickedManipulatorTest.java

```diff
@@ -0,0 +1,313 @@
+package org.mage.test.cards.single.woe;
+
+import mage.cards.Card;
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.players.Player;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+import java.util.Set;
+
+/**
+ * @author Susucr
+ */
+public class AshiokWickedManipulatorTest extends CardTestPlayerBase {
+
+    /**
+     * Ashiok, Wicked Manipulator
+     * {3}{B}{B}
+     * Legendary Planeswalker — Ashiok
+     * <p>
+     * If you would pay life while your library has at least that many cards in it, exile that many cards from the top of your library instead.
+     * +1: Look at the top two cards of your library. Exile one of them and put the other into your hand.
+     * −2: Create two 1/1 black Nightmare creature tokens with "At the beginning of combat on your turn, if a card was put into exile this turn, put a +1/+1 counter on this creature."
+     * −7: Target player exiles the top X cards of their library, where X is the total mana value of cards you own in exile.
+     * <p>
+     * Loyalty: 5
+     */
+    private static final String ashiok = "Ashiok, Wicked Manipulator";
+
+    /**
+     * Final Payment
+     * {W}{B}
+     * Instant
+     * <p>
+     * As an additional cost to cast this spell, pay 5 life or sacrifice a creature or enchantment.
+     * <p>
+     * Destroy target creature.
+     */
+    private static final String finalPayment = "Final Payment";
+
+    /**
+     * Well sometimes you do need a 2/2 vanilla with mana value 2.
```

Comment:
Silvercoat Lion over Runeclaw Bear, huh?

Reply:
More coverage of all 2/2 for 2 is good